includes/widgets/image-carousel.php

```diff
@@ -931,26 +951,43 @@ protected function render() {
 
 		$slides_count = count( $settings['carousel'] );
 
+		if ( 'yes' ===  $settings['autoplay'] ) { ?>
+			<button class="elementor-screen-only a11y-play-button e-active" aria-label="<?php echo esc_html__( 'Pause slides', 'elementor' ); ?>">
+				<span class="a11y-play-button-start">
+					<?php Icons_Manager::render_icon(
+						[
+							'library' => 'fa-solid',
+							'value' => 'fas fa-play',
+						],
+						[ 'aria-hidden' => 'true' ]
+					); ?>
+				</span>
+				<span class="a11y-play-button-stop">
+					<?php Icons_Manager::render_icon(
+						[®ia-hidden' => 'true' ]
+					); ?>
+				</span>
+			</button>
+		<?php }
 		?>
 		<div <?php $this->print_render_attribute_string( 'carousel-wrapper' ); ?>>
 			<div <?php $this->print_render_attribute_string( 'carousel' ); ?>>
 				<?php // PHPCS - $slides contains the slides content, all the relevent content is escaped above. ?>
 				<?php echo implode( '', $slides ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>
 			</div>
 			<?php if ( 1 < $slides_count ) : ?>
-				<?php if ( $show_dots ) : ?>
-					<div class="swiper-pagination"></div>
-				<?php endif; ?>
 				<?php if ( $show_arrows ) : ?>
 					<div class="elementor-swiper-button elementor-swiper-button-prev" role="button" tabindex="0">
 						<?php $this->render_swiper_button( 'previous' ); ?>
-						<span class="elementor-screen-only"><?php echo esc_html__( 'Previous image', 'elementor' ); ?></span>
```

Comment:
These screen readers notifications have been replaced by a standard functionality from the SwiperJS library.